packages/expo-updates/android/src/main/java/expo/modules/updates/UpdatesUtils.kt

```diff
@@ -28,6 +28,8 @@ object UpdatesUtils {
   private val TAG = UpdatesUtils::class.java.simpleName
 
   private const val UPDATES_DIRECTORY_NAME = ".expo-internal"
+  private const val ANDROID_EMBEDDED_URL_BASE_ASSET = "file:///android_asset/"
+  private const val ANDROID_EMBEDDED_URL_BASE_RESOURCE = "file:///android_res/"
```

Comment:
Isn’t this already defined in one of the expo-asset classes, so we don’t have to duplicate the constant here?

Reply:
expo-updates doesn't have dependency to expo-asset so it's not really possible to share the constants between them.